autoreject/utils.py

```diff
@@ -461,3 +460,17 @@ def _fast_map_meg_channels(info, pick_from, pick_to,
     mne.set_log_level(verbose)
 
     return fmd['data']
+
+
+def _my_setup_dots(mode, info, coils, ch):
+    """
+    Monkey patcher for MNE's _setup_dots that changed parametrization
+    in version 0.24
+    """
```

Comment:
Can you make this a one line docstring like in the rest of the repo?

Reply:
done, thx